.github/scripts/generate_binary_build_matrix.py

```diff
@@ -187,7 +141,7 @@ def arch_type(arch_version: str) -> str:
 
 WHEEL_CONTAINER_IMAGES = {
     **{
-        gpu_arch: f"pytorch/manylinux2_28-builder:cuda{gpu_arch}-{DEFAULT_TAG}"
+        gpu_arch: f"308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/manylinux2_28-builder-cuda{gpu_arch}:6b70660514c75c7ea9d0da1bb5925c7513dce353"
```

Comment:
TODO: This will be removed before pushing

Reply:
Why don't you want to remove this one now?

Reply:
Will keep for now. To make sure signal is green until is approved.